extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -1,26 +1,23 @@
 <?php
 
 class CommunityPageSpecialInsightsModel {
-	const INSIGHTS_MODULE_ITEMS = 5;
-	const INSIGHTS_CONFIG_SORT_TYPE_KEY = 'sortingType';
-	const INSIGHTS_CONFIG_PAGEVIEWS_KEY = 'displayPageviews';
+	const INSIGHTS_MODULE_ITEMS = 3;
+	const INSIGHTS_ICON = 'icon';
+	const INSIGHTS_TITLE = 'title';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)